website_and_docs/content/documentation/selenium_manager.en.md

```diff
@@ -92,11 +104,13 @@ The following table summarizes all the supported arguments supported by Selenium
 In addition to the configuration keys specified in the table before, there are some special cases, namely:
 
 - Browser version. In addition to `browser-version`, we can use the specific configuration keys to specify custom versions per supported browser. This way, the keys `chrome-version`, `firefox-version`, `edge-version`, etc., are supported. The same applies to environment variables (i.e., `SE_CHROME_VERSION`, `SE_FIREFOX_VERSION`, `SE_EDGE_VERSION`, etc.).
-- Driver version. Following the same pattern, we can use `chromedriver-version`, `geckodriver-version`,  `msedgedriver-version`,  etc. (in the configuration file), and `SE_CHROMEDRIVER_VERSION`, `SE_GECKODRIVER_VERSION`, `SE_MSEDGEDRIVER_FIREFOX`,  etc. (as environment variables).
+- Driver version. Following the same pattern, we can use `chromedriver-version`, `geckodriver-version`,  `msedgedriver-version`,  etc. (in the configuration file), and `SE_CHROMEDRIVER_VERSION`, `SE_GECKODRIVER_VERSION`, `SE_MSEDGEDRIVER_VERSION`,  etc. (as environment variables).
```

Comment:
I kind of think we should merge browser version and driver version before we get out of beta, but that's a discussion for another time. 😄

Reply:
Browser versions and driver versions are different things. Maybe similar in the case of Chrome and Edge since they follow the same versioning schema for browser and driver, but not Firefox (e.g., Firefox 116 needs geckodriver 0.32.0).